reference-data/groups/faculty.json

```diff
@@ -1,5 +1,6 @@
 {
   "id" : "503a81cd-6c26-400f-b620-14c08943697c",
   "group":"faculty",
-  "desc":"Faculty Member"
+  "desc":"Faculty Member",
+  "expirationOffset": 365
```

Comment:
Should the reference groups have these values in existing systems?

If so, please write an upgrade script for the existing records.

Reply:
No, as the value is optional, there is no need for existing systems to have these values.
I added them to the reference-data to show usage in reference-data resp sample-data.